README.md

```diff
@@ -1,68 +1,86 @@
+ 
+ 
+## Prerequisites
+ 
+-   [Adyen test account](https://docs.adyen.com/get-started-with-adyen)
+-   [API key](https://docs.adyen.com/development-resources/api-credentials#generate-api-key). For testing, your API credential needs to have the [API PCI Payments role](https://docs.adyen.com/development-resources/api-credentials#roles).
 - Python 2.7 or 3.6
 - Packages: requests or pycurl ( optional )
-- Adyen account. If you don't have this you can request it here: https://www.adyen.com/home/discover/test-account-signup#form
+ 
 
-## Installation
+ ## Installation
 
 ### For development propose
 
-Clone this repository and run ```make install```
+Clone this repository and run 
+~~~~ bash 
+make install
+~~~~
 
 ### For usage propose
 
-Use pip command: ```pip install Adyen```
-
-## Usage
-
-Create a class instance of the 'Adyen' class.
-
-```python
+Use pip command: 
+~~~~ bash 
+pip install Adyen
+~~~~
+
+## Using the library
+ 
+ 
+### General use with API key
+ 
+~~~~ python
 import Adyen
 
 ady = Adyen.Adyen()
 
-ady.payment.client.username = "webservice user name"
+ady.payment.client.xapikey = "YourXapikey"
 ady.payment.client.skin_code = "skin code for Hosted Payment pages"
 ady.payment.client.hmac = "HMAC key for skin code"
 ady.payment.client.platform = "test" # Environment to use the library in.
 ady.payment.client.merchant_account = "merchant account name from CA"
-ady.payment.client.password = "webservice user password"
-```
-
-## Documentation
-* https://docs.adyen.com/developers/development-resources/libraries
-* https://docs.adyen.com/developers/checkout
+~~~~
+ 
+### Example integration
+ 
+For a closer look at how our python library works, clone our [example integration](https://github.com/adyen-examples/adyen-python-online-payments). This includes commented code, highlighting key features and concepts, and examples of API calls that can be made using the library.
 
-## Support
-If you have a feature request, or spotted a bug or a technical problem, create a GitHub issue. For other questions, contact our [support team](https://support.adyen.com/hc/en-us/requests/new?ticket_form_id=360000705420).
 
 ## Contributing
-We strongly encourage you to join us in contributing to this repository so everyone can benefit from:
-* New features and functionality
-* Resolved bug fixes and issues
-* Any general improvements
-
-Read our [**contribution guidelines**](CONTRIBUTING.md) to find out how.
-
+ 
+We encourage you to contribute to this repository, so everyone can benefit from new features, bug fixes, and any other improvements.
+ 
+ 
+Have a look at our [contributing guidelines](https://github.com/Adyen/adyen-python-api-library/blob/develop/CONTRIBUTING.md) to find out how to raise a pull request.
+ 
+ 
+## Support
+If you have a feature request, or spotted a bug or a technical problem, [create an issue here](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2FAdyen%2Fadyen-web%2Fissues%2Fnew%2Fchoose).
```

Comment:
Is this the correct link? I think this is a login + redirect , but the issue form alone should already do the login check